libraries/Microsoft.Bot.Builder/Adapters/TestAdapter.cs

```diff
@@ -445,6 +445,7 @@ public Activity MakeActivity(string text = null)
                 Text = text,
             };
 
+            Thread.CurrentThread.CurrentCulture = new CultureInfo(activity.Locale);
```

Comment:
> Thread.CurrentThread.CurrentCulture = new CultureInfo(activity.Locale); [](start = 11, length = 72)

No this is not the right place to do this.

Reply:
changed